app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -2,7 +2,11 @@
   <div class="registration crayons-card">
     <div class="registration__content">
       <h1 class="registration__title">
-        Welcome to <%= community_name %>
+        <% if params[:state] == "new-user" && invite_only_mode? %>
+          <%= community_name %> is in invitation only mode.
```

Comment:
This overall looks good to me. The only change I immediately feel is that this phrase is a little off because being in a "mode" is more insider terminology to the admin vs the end user.

```suggestion
          <%= community_name %> is invite only.
```

Reply:
Non-blocking comment: should we include some text along the lines of, "You will need an admin of <community name> to send you an invitation to join this community"?

Reply:
I think ultimately this design/copy should depend on use-cases... Essentially how the forem is marketing itself... And how the invitation system works. I think there _could_ be a type of forem which is invite-only but users themselves can send out a certain amount of invitations.

Reply:
Yeah, I didn't want to make any design decisions here because I know it will change as soon as we get some user feedback. Ben's change is good with me.

Reply:
Need to fix the tests though!